contentgrid-spring-data-rest/src/main/java/com/contentgrid/spring/data/rest/webmvc/DefaultDomainTypeToHalFormsPayloadMetadataConverter.java

```diff
@@ -80,7 +80,7 @@ public PayloadMetadata convertToUpdatePayloadMetadata(Class<?> domainType) {
         )
                 .filter(property -> !Objects.equals(property.getInputType(), HtmlInputType.URL_VALUE))
                 .forEachOrdered(properties::add);
-        return new ClassnameI18nedPayloadMetadata(domainType, properties);
+        return new PayloadMetadataAndMediaType(new ClassnameI18nedPayloadMetadata(domainType, properties), getCreatePayloadMediaType());
```

Comment:
This is not the correct mediatype for the update form?

The update form is always json, multipart upload is not supported for updates?